.env

```diff
@@ -1,6 +1,6 @@
 WORDPRESS_VERSION=6.0.0
 WOOCOMMERCE_VERSION=5.8.0
-PHP_VERSION=7.3
+PHP_VERSION=7.4
```

Comment:
Why do we need php 7.4? I'm afraid that if we start testing with 7.4 we will miss that it's broken on 7.3 ([we require 7.3](https://github.com/storekeeper-company/storekeeper-woocommerce-b2c/blob/master/composer.json#L33))

Reply:
Reeverted to  7.3, tested as well.